packages/suite/src/hooks/suite/useCoinjoinAccountInfo.ts

```diff
@@ -0,0 +1,30 @@
+import { useMemo } from 'react';
+import { selectAccountByKey } from '@suite-common/wallet-core';
+import { selectCoinjoinAccountByKey } from '@wallet-reducers/coinjoinReducer';
+import { breakdownCoinjoinFunds, CoinjoinFundsBreakdown } from '@wallet-utils/coinjoinUtils';
+import { useSelector } from './useSelector';
+
+export const useCoinjoinAccountInfo = (accountKey: string) => {
```

Comment:
Don't abuse hooks, this should be selector :D It will be much more versatile then and could also be used in thunks etc.

Reply:
You can use selector in this hook if you want to add some useEffect from example. Also it should be one line of code too: `const accountCoinjoinInfo = useSelector(selectAccountCoinjoinInfo())`

Reply:
Let's try it as a selector, you are right! ;) We need to transition to new patterns and more complex selectors.

2e0f0a4b8f28966b3a4c42cd89ed42a81a630a40